cmd/blockchaincmd/deploy.go

```diff
@@ -203,6 +205,7 @@ so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.
 
 	cmd.Flags().BoolVar(&partialSync, "partial-sync", true, "set primary network partial sync for new validators")
 	cmd.Flags().Uint32Var(&numNodes, "num-nodes", constants.LocalNetworkNumNodes, "number of nodes to be created on local network deploy")
+	cmd.Flags().StringVar(&validatorManagerAddress, "validator-manager-address", validatorManagerSDK.ProxyContractAddress, "validator manager address")
```

Comment:
let's make deploy to not have this flag but use the default address. what is blockchain going to do if given
another address? nothing will work because:
1) currently the manager is fixed on genesis
2) in the future the manager will be deployed by this command, and the address will be generasted automatically.

let's make this command add the default manager address to the sidecar, so the other commands can refer to it